### PR TITLE
[SPARK-144] Give MechaClient a persistent Galaxy object

### PR DESCRIPTION
**Pull request by Eearslya**

### CODE REVIEW

File: tests/unit/test_mechaclient.py [hunk 1]
```diff
@@ -47,6 +48,13 @@ def test_verify_rat_board_private(bot_fx):
     assert bot_fx._rat_board is None
 
 
+def test_verify_galaxy_private(bot_fx):
+    """
+    Asserts the private reference for the Galaxy is none, after instantiation.
+    """
+    assert bot_fx._galaxy is None
```
Comment (theunkn0wn1) [inline tests/unit/test_mechaclient.py]: [error] this test is unnecessary, as it tests python's behavior rather than our code specifically.

**Review by theunkn0wn1 — CHANGES_REQUESTED** [review]
one small testing related issue, otherwise LGTM

File: tests/unit/test_mechaclient.py (hunk 1 above)
Comment (Eearslya) [reply tests/unit/test_mechaclient.py]: May need to eventually look at the other 4 tests in there as well. I was simply following their example.

Comment (theunkn0wn1) [reply tests/unit/test_mechaclient.py]: Might be wise. guess we need to schedule a review at some point 😕

**Review by theunkn0wn1 — APPROVED** [review]

**Pull request merged**